### PR TITLE
Added missing mockery dev dependency

### LINKED ISSUE #78: Reminder: Add mockery to require-dev

### PR DESCRIPTION
**Pull request by prisis**
fixes #78

### CODE REVIEW

**Comment by prisis** [comment]
Don't merge it, tests a not working...

File: composer.json [hunk 1]
```diff
@@ -24,15 +24,16 @@
...
         "psr/cache": "~1.0",
         "cache/tag-interop": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.0|^5.0",
+        "phpunit/phpunit": "^4.0 || ^5.0",
         "cache/cache": "dev-master",
         "symfony/cache": "^3.1",
         "madewithlove/illuminate-psr-cache-bridge": "^1.0",
+        "mockery/mockery": "^0.9.9",
```
Comment (Nyholm) [inline composer.json]: Where do we use Mockery?

Comment (prisis) [reply composer.json]: https://travis-ci.org/php-cache/integration-tests/jobs/200021430 
and scroll down

Comment (prisis) [reply composer.json]: 4) Cache\Bridge\SimpleCache\Tests\SimpleCacheBridgeTest::testFetch
Error: Class 'Mockery' not found

**Comment by Nyholm** [comment]
Do we use mockery in these tests?

**Comment by prisis** [comment]
We using Mockery in our PHPCache tests

**Comment by Nyholm** [comment]
Can you direct me to a row? I cannot find it. 

Sorry.

**Comment by prisis** [comment]
It only happens on my local machine -.- hmmm

**Comment by prisis** [comment]
@Nyholm found one https://travis-ci.org/php-cache/integration-tests/jobs/206611026

**Comment by Nyholm** [comment]
Ah okey. We need it for the simple cache bridge. 

Thanks!

**Pull request merged**